src/panels/MailPanel.php

```diff
@@ -108,7 +108,7 @@ public function getSummary()
     {
         return Yii::$app->view->render('panels/mail/summary', [
             'panel' => $this,
-            'mailCount' => count($this->data),
+            'mailCount' => is_array($this->data) ? count($this->data) : 0,
```

Comment:
If I understand this PR right, a mail is sent, but it serialization fails, so the panel gets broken.
In that case, isn't misleading for the developer to display 0?

Reply:
Maybe we should display something else than 0? Something like ⚠️?

Reply:
Then it no longer is a number, meaning other code might break.
We could do -1. Note that for on mail panel itself it will show a stack trace of the error anyway.

Reply:
-1 sounds better for the case.

Reply:
I've checked and it seems using ⚠️ is fine in this case.